go/v1/statement_test.go

```diff
@@ -39,18 +40,46 @@ func createTestStatement(t *testing.T) *Statement {
 	}
 }
 
-func TestJsonUnmarshalStatement(t *testing.T) {
+func TestProtojsonUnmarshalStatement(t *testing.T) {
 	var wantSt = `{"_type":"https://in-toto.io/Statement/v1","subject":[{"name":"theSub","digest":{"alg1":"abc123"}}],"predicateType":"thePredicate","predicate":{"keyObj":{"subKey":"subVal"}}}`
 
 	got := &Statement{}
 	err := protojson.Unmarshal([]byte(wantSt), got)
 	assert.NoError(t, err, "error during JSON unmarshalling")
 
 	want := createTestStatement(t)
-	assert.NoError(t, err, "unexpected error during test Statement creation")
```

Comment:
I was confused about this line removal, but I see that this error is already checked above 👍🏻

Reply:
Yeah, removed the redundant checks 😅